src/js/angular/clustermanagement/controllers/cluster-management.controller.js

```diff
@@ -177,24 +162,7 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
     };
 
     $scope.showCreateClusterDialog = function () {
```

Comment:
You can remove this function, and where it is used, simply call $scope.showUpdateClusterGroupDialog() directly.

Reply:
done